pkg/volume/azure_file/azure_provision.go

```diff
@@ -189,6 +193,11 @@ func (a *azureFileProvisioner) Provision(selectedNode *v1.Node, allowedTopologie
 		return nil, fmt.Errorf("claim.Spec.Selector is not supported for dynamic provisioning on Azure file")
 	}
 
+	tags, err := azure_dd.ConvertTagsToMap(customTags)
```

Comment:
move this `ConvertTagsToMap` to a common place, e.g. in Azure cloud provider lib?

Reply:
done